workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/CodeGeneration/Model/SchemaBundleV1/Common.cs

```diff
@@ -1,13 +1,73 @@
+using System;
 using System.Collections.Generic;
+using System.Linq;
 using Newtonsoft.Json;
 
 namespace Improbable.Gdk.CodeGeneration.Model.SchemaBundleV1
 {
-    public class Identifier
+    public class Identifier : IEquatable<Identifier>
     {
         [JsonProperty("qualifiedName")] public string QualifiedName;
         [JsonProperty("name")] public string Name;
         [JsonProperty("path")] public List<string> Path;
+
+        public string PackagePath => string.Join(".", Path.Take(Path.Count - 1));
+
+        public static Identifier FromQualifiedName(string qualifiedName)
+        {
+            return new Identifier
+            {
+                QualifiedName = qualifiedName,
+                Name = qualifiedName.Split(".").Last(),
+                Path = qualifiedName.Split(".").ToList()
+            };
+        }
+
+        public bool IsChild(Identifier identifier)
```

Comment:
Does `a.isChild(b)` means that `a` is child of `b` of that `b` is child of `a`?
For me it reads like in this function it is latter, though semantically I would expect this to be the former, or am I reading this wrong?